<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pagina_registro</class>
 <widget class="QWidget" name="pagina_registro">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="logo_app">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>261</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>110</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 35pt &quot;Lucida Grande&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Registro</string>
   </property>
  </widget>
  <widget class="QPushButton" name="boton_login">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>530</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>true</bold>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 15px;  /* Bordes redondeados */
border: 2px solid;  /* Borde del botón */
padding: 10px 20px;  /* Espaciado interno */
font-size: 16px;  /* Tamaño de la fuente */
</string>
   </property>
   <property name="text">
    <string>Iniciar Sesión</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="password_1">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>280</y>
     <width>250</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 5px;  /* Bordes redondeados */</string>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>250</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;Lucida Grande&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Contraseña:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="boton_registro">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>440</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 15px;  /* Bordes redondeados */
border: 2px solid;  /* Borde del botón */
padding: 10px 20px;  /* Espaciado interno */
font-size: 16px;  /* Tamaño de la fuente */
</string>
   </property>
   <property name="text">
    <string>Registrarse</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>180</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;Lucida Grande&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Correo electrónico:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="correo">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>210</y>
     <width>250</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 5px;  /* Bordes redondeados */</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>500</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 15pt &quot;Lucida Grande&quot;;
color: rgb(235, 247, 78);
</string>
   </property>
   <property name="text">
    <string>¿Ya cuentas con una cuenta?</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>320</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;Lucida Grande&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Confirmar contraseña:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="password_2">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>350</y>
     <width>250</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 5px;  /* Bordes redondeados */</string>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>180</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;Lucida Grande&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Nombre(s):</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nombre">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>210</y>
     <width>250</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 5px;  /* Bordes redondeados */</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="apellido">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>280</y>
     <width>250</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 5px;  /* Bordes redondeados */</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>250</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;Lucida Grande&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Apellidos:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>320</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;Lucida Grande&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Teléfono:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="telefono">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>350</y>
     <width>250</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 5px;  /* Bordes redondeados */</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>410</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: &quot;Lucida Grande&quot;;
color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
